src/error.rs

```diff
@@ -81,6 +87,9 @@ impl fmt::Display for Error {
             Error::StackOverflow => write!(f, "Max stack size exceeded for backtracking"),
             Error::BacktrackLimitExceeded => write!(f, "Max limit for backtracking count exceeded"),
             Error::__Nonexhaustive => unreachable!(),
+            Error::InvalidGroupName => write!(f, "Could not parse group name"),
+            Error::InvalidGroupNameBackref(s) => write!(f, "Invalid group name in back reference: {}", s),
+            Error::NamedBackrefOnly => write!(f, "numbered backref/call is not allowed. (use name)"),
```

Comment:
Style doesn't match other messages, let's change to `Numbered backref/call not allowed because named group was used, use a named backref instead`

(Also makes it clearer.)